### DISCUSSION TITLE
Is there a way to disallow pull requests with empty descriptions?

### DISCUSSION THREAD

**Discussion by Mr0grog** [community/community#22282] [New to GitHub]
<p>On open-source projects (well, really all projects), it’d be really helpful to disallow proposing pull requests with empty descriptions. I figured that would be a setting alongside other pull request/branch protection features, but it doesn’t seem to be, and a search in help site and in this forum don’t seem to turn up anything about it either. Is there a way to do this that I’m missing? Otherwise, I’d <em>really</em> love to see this feature built in.</p>
<p>I could probably write a bot or app that could automatically post a negative review or comment on these PRs, but:</p>
<ul>
<li>It would be a lot more straightforward if the author got a gentle reminder right when they try to post the PR that they should add a description, rather than getting a message after the fact.</li>
<li>Not having to create a public trail of nanny-style comments or negative reviews seems like it would be a lot more polite :slight_smile:
</li>
</ul>

**Comment by mpboom** [comment, marked as answer]
<p>I don’t think this is natively possible right now, so it should be logged as a feature request soon.</p>
<p>However, personally I don’t think using a bot for now is wrong. If you make sure to clearly state in your contributing guidelines that you do not wish to have any empty commit messages it should be fine.</p>

**Comment by sim642** [comment]
This is very relevant again, as Copilot seems to like creating empty "Initial plan" commits without any possible override: https://github.com/orgs/community/discussions/178992.
